joomla/joomla-cms
# [4.0] Namespace Editors - TinyMCE plugin

### Summary of Changes

Namespaces Editors - TinyMCE plugin.

### Testing Instructions

Delete `libraries/autoload_psr4.php` file.
Edit `Editor - TinyMCE` plugin options.
Check that TinyMCE builder appears.
Check that `Site Skin` and `Administrator Skin` selectors appear correctly inside the builder.
Check that `Images Directory` selector appears correctly.

### Documentation Changes Required

IDK.
